src/4.3_to_5.0/accumulateMigratedKpiTitles.ts

```diff
@@ -107,35 +113,44 @@ export function getMigratedKpiTitles(
   const ordinalFields = [...(mapping.columns ?? []), ...(mapping.rows ?? [])];
 
   legacyTitles.forEach(({ title, tuple }) => {
-    const memberUniqueNames: string[] = [];
+    try {
+      const memberUniqueNames: string[] = [];
 
-    ordinalFields.forEach((field) => {
-      if (field.type === "hierarchy") {
-        const hierarchyUniqueName = quote(
-          field.dimensionName,
-          field.hierarchyName,
-        );
-        const namePath = tuple[hierarchyUniqueName];
-        if (namePath) {
-          memberUniqueNames.push(
-            `${hierarchyUniqueName}.${quote(...namePath)}`,
+      ordinalFields.forEach((field) => {
+        if (field.type === "hierarchy") {
+          const hierarchyUniqueName = quote(
+            field.dimensionName,
+            field.hierarchyName,
           );
+          const namePath = tuple[hierarchyUniqueName];
+          if (namePath) {
+            memberUniqueNames.push(
+              `${hierarchyUniqueName}.${quote(...namePath)}`,
+            );
+          }
         }
+      });
+
+      if (measuresPositionInTuple > -1) {
+        const measureName = tuple[quote("Measures", "Measures")][0];
+        memberUniqueNames.splice(
+          measuresPositionInTuple,
+          0,
+          quote("Measures", measureName),
+        );
       }
-    });
-
-    if (measuresPositionInTuple > -1) {
-      const measureName = tuple[quote("Measures", "Measures")][0];
-      memberUniqueNames.splice(
-        measuresPositionInTuple,
-        0,
-        quote("Measures", measureName),
-      );
-    }
 
-    const tupleKey = memberUniqueNames.join(",");
-    migratedTitles[tupleKey] = title;
+      const tupleKey = memberUniqueNames.join(",");
+      migratedTitles[tupleKey] = title;
+    } catch (error) {
+      errors.push(error as Error);
+    }
```

Comment:
Again, looks like a big change but I just wrapped this logic block in a try-catch.